guides/deployments/postgres.md

```diff
@@ -9,41 +9,40 @@ database.
 ## Background
 
 For convenience and ease of installation, Coder's default Helm Chart settings
-will  [PostgreSQL database](https://www.postgresql.org/) within the
+will deploy a [PostgreSQL database](https://www.postgresql.org/) within the
 installation's Kubernetes namespace. This is useful for evaluation purposes;
-however, we **recommend using an out-of-cluster database for production**, to
+however, we **recommend using an out-of-cluster database for production** to
 streamline maintenance operations, such as backups and upgrades. The database
-container is not backed up and will be lost when deleting the namespace or
-Kubernetes cluster.
+state is *not* backed up and will be lost when deleting the Kubernetes namespace
+or cluster.
 
 > For optimal performance, it is important to ensure that the round-trip latency
 > between the Coder control plane services and the database is low. We recommend
-> ensuring that the database is within the same datacenter as the control plane,
-> such as within the same cloud availability zone.
+> ensuring that the database is within the same data center as the control
+> plane, such as within the same cloud availability zone. If the latency exceeds
+> 10ms under load, users may observe significantly degraded experience.
```

Comment:
```suggestion
> plane, such as within the same cloud availability zone.
```

Reply:
Isn't it useful to have some sort of criteria so that people know what "a lot" looks like?

Reply:
We don't give any instructions for how to calculate "latency under load" so we should avoid giving them numbers like this. I think the rest of the paragraph does a good enough job of conveying that the latency should be in the low milliseconds or sub-millisecond range.

Reply:
I dunno, that seems like a little too read-between-the-lines thing to me, but I don't feel strongly enough about this to hold up this PR, will remove that line